### PR TITLE
Remove Gemini instructions for finding nodes in Fedora

### PR DESCRIPTION
**Pull request by rosiel**
## Purpose / why

Thanks for the catch by Ian Stewart on Slack: https://islandora.slack.com/archives/CM5PPAV28/p1632314232177500

If there's a ticket for "Remove Gemini" this is related, sorry I can't find it. 

## What changes were made?

* Removed the section on finding the URI in the gemini databasee
* Added a small note (for deprecation purposes) mentioning that this used to be done by Gemini
* Re-worded the description of the "Fedora URI Pseudofield" to reflect my current understanding of 
  * Whether the Fedora URI still means that the object is in Fedora (since I can see them on a site where Fedora has never been working, no, it doesn't. This is worth mentioning because this field was created (at least partially) in response to a call (at least partially by me) to "know whether my Drupal thing has been synced to Fedora", and the presence of that URI was the shibboleth. Now we have to open the URI to find out.)
  * Where that URI is computed (if I recall, it's Milliner)

## Verification

Need someone with familiarity with the code to double-check. Is it Milliner that computes this?

## Interested Parties

(do you know if Ian has a Github?)

@Islandora/8-x-committers

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?
* [ ] Does this PR update the _last updated on_ date on the documentation page?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

File: docs/technical-documentation/testing-notes.md [hunk 1]
```diff
@@ -1,43 +1,12 @@
...
 
-For more information about the Gemini service, see the [Gemini README](https://github.com/Islandora/Crayfish/tree/main/Gemini).
+### Enabling the Fedora URI Pseudo-field
 
-## Fedora URI Pseudo-field
+To display the Fedora URI pseudo-field on a Drupal node, media, or taxonomy term, go to Manage > Configuration > Islandora and select all the bundles for which you would like the Fedora URI displayed. Once you have selected the bundles, and cleared the cache, the new pseudo-field will appear at the bottom of _all display modes_. You can alter where in the display the Fedora URI field appears, by going to the "Manage Display" page for the bundle. For example, for a Repository Item, you'd go to http://localhost:8000/admin/structure/types/manage/islandora_object/display. In that list you should see `Fedora URI` which you can move around (or hide) as desired. Clearing cache may be necessary to refresh the node display.
```
Comment (eldonquijote) [inline docs/technical-documentation/testing-notes.md]: Manage > Configuration > Islandora > Core Settings and in the section "Fedora URL Display"

Also, for consistency I'd use the same 'breadcrumb' navigation style for directing users to the Manage Display page of the example: Manage > Structure > Content Types > Select "Manage Display" from drop down menu for Repository Item.

Comment (rosiel) [reply docs/technical-documentation/testing-notes.md]: Thanks! Done.

**Comment by rosiel** [comment]
@Islandora/8-x-committers 

I answered my own question, as the code to compute the Fedora URI is in Crayfish-Commons. Can anyone give this PR a second look?

**Review by dannylamb — APPROVED** [review]
Looks good to me @rosiel

**Pull request merged**